pressbooks/pressbooks-aldine
# Underline links on Aldine pages

This PR underlines links on Aldine pages for improved accessibility. Fix for https://github.com/pressbooks/pressbooks-aldine/issues/300

To test:
1. Checkout this branch
2. Look at links on pages throughout Aldine (including Home, Catalog, Help, About). Observe that links inside of body content are rendered with underlining, which call to action links, footer links, and catalog links are unaffected.
